io/www/src/fragment/pipeline.js

```diff
@@ -115,9 +108,19 @@ async function mainProcess(context) {
         context = { ...context, translatedId, dictionaryId };
     }
 
-    for (const transformer of [fetchFragment, translate, settings, replace, wcs, corrector]) {
-        /* istanbul ignore next */
+    for (const transformer of PIPELINE) {
+        if (transformer.init) {
+            const initContext = structuredClone(context);
```

Comment:
we can add a comment, smthg like
'structuredClone  to prevent init from modifying context'